vm/actor/src/builtin/power/state.rs

```diff
@@ -2,182 +2,100 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use super::CONSENSUS_MINER_MIN_POWER;
-use crate::{BalanceTable, BytesKey, Multimap, Set, StoragePower, HAMT_BIT_WIDTH};
+use crate::{BytesKey, Multimap, HAMT_BIT_WIDTH};
 use address::Address;
 use cid::Cid;
 use clock::ChainEpoch;
 use encoding::{tuple::*, Cbor};
+use fil_types::StoragePower;
 use ipld_blockstore::BlockStore;
 use ipld_hamt::Hamt;
 use num_bigint::biguint_ser;
-use num_traits::{CheckedSub, Zero};
+use num_traits::CheckedSub;
 use vm::{Serialized, TokenAmount};
 
 /// Storage power actor state
 #[derive(Default, Serialize_tuple, Deserialize_tuple)]
 pub struct State {
+    #[serde(with = "biguint_ser")]
+    pub total_raw_byte_power: StoragePower,
     #[serde(with = "biguint_ser")]
     pub total_network_power: StoragePower,
```

Comment:
reason this is kept? simplicity for interim changes?

Reply:
No it was overlooked. Will remove.